src/docker/compose.e2e.yml

```diff
@@ -36,6 +36,15 @@ services:
     volumes:
       - db:/var/lib/postgresql/data
 
+  diesel:
```

Comment:
How about alphabetizing the names of the services, such that this is between `chain` and `indexer`?